docs/create_allocator_service.md

```diff
@@ -0,0 +1,435 @@
+......................+++
+writing new private key to '/tmp/allocsvc.key'
+-----
+```
+
+Create a public private key pair that will be bound to the pod and used by the web server :
+```
+openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /tmp/tls.key -out  /tmp/tls.crt -subj "/CN=my-allocator-w3/O=my-allocator-w3"
+```
+
+The output should be something like:
+```
+Generating a 2048 bit RSA private key
+....................................................+++
+......................+++
+writing new private key to '/tmp/tls.key'
+-----
+```
+
+
+### 3. Create Kubernetes Secrets
+
+The allocatorsecret will allow the service to use TLS for connections with workers.
+
+Create the secret by running this command:
+```
+kubectl create secret tls allocatorsecret --cert=/tmp/allocsvc.crt --key=/tmp/allocsvc.key
+```
+
+The output should be something like:
+```
+secret "allocatorsecret" created
+```
+
+The allocatorw3secret will let data be served by the webserver over https.
+
+Create the secret by running this command:
+```
+kubectl create secret tls allocatorw3secret --cert=/tmp/tls.crt --key=/tmp/tls.key
+```
+The output should be something like:
+```
+secret "allocatorw3secret" created
+```
+
+See that the secrets exist by running:
+```
+kubectl get secrets
+```
+
+The output should contain the secrets:
+```
+NAME                     TYPE                                  DATA      AGE
+...
+allocatorsecret          kubernetes.io/tls                     2         29s
+allocatorw3secret        kubernetes.io/tls                     2         15s
+...
+```
+
+
+### 4. Create the Service Account  
+This service will interact with the Agones via the Agones API by using a service account named fleet-allocator.  Specifically, the fleet-allocator service account is granted permissions to perform create operations against FleetAllocation objects.  The complete service account, role, and role binding definitions are in agones/examples/allocator-service/service-account.yaml.
+
+Create the service account by changing directories to your local agones/examples/allocator-service directory and running this command:
+```
+kubectl create -f service-account.yaml
+```
+
+The output should look like this:
+```
+role "fleet-allocator" created
+serviceaccount "fleet-allocator" created
+rolebinding "fleet-allocator" created
+```
+
+
+### 5. Define and Deploy the Service
+The service definition defines a nodePort service which uses https, and sets up ports and names.  The deployment describes the number of replicas we would like, which account to use, which image to use, and defines a health check.  The complete service definition is in agones/examples/allocator-service/allocator-service.yaml.
+
+Define and Deploy the service by running this command:
+```
+kubectl create -f allocator-service.yaml
+```
+
+The output should look like this:
+```
+service "fleet-allocator-backend" created
+deployment "fleet-allocator" created
+```
+
+
+### 6. Define an Ingress Resource
```

Comment:
Like i said before, I think we could simplify things by removing Ingress from the example, and just using a `Service`. WDYT of that idea?

Reply:
I picked Ingress because it seemed to be the most secure way, and also the way that provides the most flexibility.  I'll read more about Services.  What I don't know how to do is use a vanilla service with tls.